ChildProject/pipelines/metricsFunctions.py

```diff
@@ -297,4 +308,13 @@ def cp_dur(annotations: pd.DataFrame, duration: int, **kwargs):
         value = can_dur / speech_dur
     else:
         value = np.nan
-    return value
\ No newline at end of file
+    return value
+
+@metricFunction(set(),{"lena_conv_turn_type"}) 
+def lena_CVC(annotations: pd.DataFrame, duration: int, **kwargs):
+    """number of child vocalizations according to LENA's extraction
+    
+    Required keyword arguments:
+    """
+    conv_types = {'TIMR', 'TIFR'}
+    return annotations[annotations["lena_conv_turn_type"].isin(conv_types)].shape[0]
```

Comment:
I wonder if our implementations of lena_CVC (mine is described [here](https://github.com/LAAC-LSCP/ChildProject/pull/376#issuecomment-1212088055)) yield the same results. I'll check early next week. This pr will have already been merged, so I'll just open an issue if the result differ.

Reply:
it would be great to check!
Here's the relevant its section:
```
                <Segment spkr="MAN" average_dB="-34.17" peak_dB="-24.66" conversationInfo="|EC|9|4|2|CIC|TIMR|FI|" maleAdultWordCnt="8.99" maleAdultNonSpeechLen="P0.00S" maleAdultUttCnt="0" maleAdultUttLen="P0.00S" startTime="PT258.38S" endTime="PT261.14S" />

```

What your pipeline is doing is extracting the numbers inside conversationInfo="|EC|9|4|2|CIC|TIMR|FI|", in particular, the number 4 here indicates that this turn has increased the CTC to 4. What we do instead is count the TIMR and TIFR, which are the codes that appear in the same conversationInfo each time LENA increases the CTC.

So it *should* be the same, but if LENA has taught me anything is humility when facing such assumptions.

Reply:
We did not check thoroughly how the lena etraction was done and we will indeed love to have your view on those metrics. Just note that here it is ` CVC` and not ` CTC` .

Reply:
I guess I can't even read :-) 

To get CTC using already extracted columns, we could count rows where `lena_block_type` is one of AICF, AICM, CIC, and XIC.

I am not sure about CVC though: 
- I am not sure counting TIFR and TIMR yields CVC,
- some .its segments contain multiple vocalizations and the `childUttCnt` attribute has to be consulted in this case.

Here is an example that illustrates both points:

```<Segment spkr="CHN" average_dB="-31.94" peak_dB="-22.55" conversationInfo="|BC|19|1|0|CM|NT|FI|" childUttCnt="2" childUttLen="P0.34S" startUtt1="PT1182.79S" endUtt1="PT1183.05S" startUtt2="PT1183.40S" endUtt2="PT1183.48S" childCryVfxLen="P0.00S" startTime="PT1182.79S" endTime="PT1183.62S" />```